lib/normalize.js

```diff
@@ -3,10 +3,27 @@ const { glob } = require('glob')
 const normalizePackageBin = require('npm-normalize-package-bin')
 const normalizePackageData = require('normalize-package-data')
 const path = require('path')
+const log = require('proc-log')
+const git = require('@npmcli/git')
 
-const normalize = async (pkg, { strict, steps }) => {
+// Replace with https://github.com/npm/git/pull/135 once it lands
```

Comment:
This landed

Reply:
Added a commit replacing this with `@npmcli/git@4.1.0`.